install.sh

```diff
@@ -46,6 +47,36 @@ is_command_present() {
     type "$1" >/dev/null 2>&1
 }
 
+dist_id=""
+dist_version=""
+
+# Check if the command is available
+if is_command_present lsb_release; then
+    dist_id=$(lsb_release -is)
+    dist_version=$(lsb_release -rs)
+elif  [ -f /etc/os-release ]; then
+    . /etc/os-release
+    dist_id=$NAME
+    dist_version=$VERSION_ID
+elif is_command_present distro; then
+    dist_id=$(distro | awk '{print $1}')
+    dist_version=$(distro | awk '{print $2}')
+else 
+    echo "Couldn't determine the OS distribution"
```

Comment:
I'm getting this message on mac, but then installation (with docker) works. Is this just for linux flavors?

Reply:
Yes, that is just for Linux flavors. I am using the  dist_id which is usually `ubuntu, rhel, fedora, centos`, etc, to download the appropriate docker repo.